src/unlimited.cpp

```diff
@@ -1491,6 +1491,45 @@ extern UniValue getsizes(const UniValue& params, bool fHelp)
 #ifdef DEBUG_LOCKORDER
     ret.push_back(Pair("lockorders",lockorders.size()));
 #endif
+
+    LOCK(cs_vNodes);
+    std::vector<CNode*>::iterator n;
+    ret.push_back(Pair("vNodes",vNodes.size()));
+    uint64_t totalThinBlockSize = 0;
+    int disconnected = 0;  // watch # of disconnected nodes to ensure they are being cleaned up
+    for (n = vNodes.begin(); n != vNodes.end(); ++n)
+    {
```

Comment:
You can reduce the lexical noise somewhat from `(*n)->..` to `n.` by declaring it as a `CNode &`.